feature/bgp/policybase/otg_tests/aspath_and_community_test/aspath_and_community_test.go

```diff
@@ -273,6 +276,10 @@ func TestCommunitySet(t *testing.T) {
 	bs := cfgplugins.NewBGPSession(t, cfgplugins.PortCount2, nil)
 	bs.WithEBGP(t, []oc.E_BgpTypes_AFI_SAFI_TYPE{oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST, oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST}, []string{"port2"}, true, true)
 
+	if deviations.BgpAspathsetUnsupported(bs.DUT) {
+		testResults[4] = true
```

Comment:
see above, we need some alternative configuration/policy to achieve the desired behavior of matching on as paths.

Reply:
Moved TC 7.4 to pull #3607 with correct resolution